WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+
+#### Presence of Mind
+*10th-level Arcane feature* <br>
+Whenever you create a spell slot from sorcery points you also gain an arcane charge of equivalent level and you can use it as part of the same bonus action.
+
+#### Savant
+*10th-level Arcane feature* <br>
+You can change your list of prepared mage spells when you finish a short or a long rest.
+
+#### Time Warp
+*14th-level Arcane feature* <br>
+You can exert your control over magic to warp time itself. When a creature you can see within 60 feet of you begins their turn, you can use your reaction to allow that creature to take another action during that turn. When you do so, you gain one level of exhaustion.
+
+### Study of Fire
+While any worthy mage is an unparalleled expert in the use of magic and heavily dedicated to their craft, those who master the forces of fire tend to be a little more audacious than their peers. These mages take pride, even pleasure in igniting their enemies in wild bursts of flame.
+
+#### Bonus Cantrip
+*2nd-level Fire feature* <br>
+When you choose this magical study at 2nd level, you learn the *control flames* cantrip. If you already know this cantrip, you learn a different wizard cantrip of your choice. The cantrip doesn't count against your number of cantrips known.
+
+<div class='footnote'>PART 1 | CLASSES</div>
+<img src='https://www.gmbinder.com/images/u17sxuH.jpg' style='position:absolute; top:580px; right:260px; width:700px' />
+<img src='https://www.gmbinder.com/images/fnPL7gX.png' style='position:absolute; top:0px; right:-120px; width:950px' />
+
+\pagebreakNum
+
+#### Hot Streak
+*2nd-level Fire feature* <br>
+The blazing energy of your spells intensifies. When you roll damage for a spell and roll the highest number possible on any of the dice, choose one of those dice, roll it again and add that roll to the damage. You can use the feature only once per turn.
+
+At 10th level, you may choose any number of dice that rolled the highest number possible, you can only reroll each dice once.
+
+#### Cauterize
+*6th-level Fire feature* <br>
+If you are reduced to 0 hit points, you can use your reaction to draw on the fiery energy that burns within you. You are instead reduced to 1 hit point, and each creature within 10 feet of you takes fire damage equal to half your mage level + your Intelligence modifier.
+
+Once you use this feature, you can’t use it again until you finish a long rest.
+
+#### Ignite
+*10th-level Fire feature* <br>
+When you deal fire damage with a spell of 1st level or higher, you can spend up to 3 sorcery points to empower a fire damage dealing spell of up to 5th-level. For each sorcery point spent you can add an extra die to a damage roll of a spell that deals fire damage.
+
+For example, when you cast Burning Hands and spend 2 points on it you add an additional 2d6 to the spells normal damage. If you where to cast Scorching Ray you can spend points on every ray individually.
```

Comment:
The second part feels like it speaks solely about Scorching Ray, and not as a general rule for spells like Scorching Ray. Could that be improved somehow?

Reply:
Is this better?
 
 If you where to cast a spell like Scorching Ray that uses multiple attacks you can spend points on every attack individually.

Reply:
I think personally, maybe something like ...

_When you cast a spell with multiple attack rolls, such as Scorching Ray, you can spend points on each attack individually._

Feels a bit right here to start both these examples the same way. Either both "If you" or "When you". Up to you which one honestly. 👍